static/js/browser2/weather_chart_section.tsx

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Component for rendering charts for various weather related properties.
+ */
+
+import React from "react";
+import axios from "axios";
+import _ from "lodash";
+import { ObservationChart } from "./observation_chart";
+
+const WEATHER = [
```

Comment:
please be more specific with the variable names. these are weather_property_names?

Reply:
done